cms/blocks.py

```diff
@@ -80,3 +80,16 @@ class AlumniBlock(blocks.StructBlock):
     quote = blocks.RichTextBlock(
         help_text="The quote that appears on the alumni section."
     )
+
+
+class LearningResourceBlock(blocks.StructBlock):
+    """
+    Block class that contains learning resources
+    """
+
+    heading = blocks.CharBlock(
+        max_length=100, help_text="The heading to display for this section on the page."
+    )
+    links = blocks.RichTextBlock(
+        help_text="The resources with links to display in this section."
```

Comment:
`Represents resources with links to display. Make/Add each links in new line of editor. `